clang/lib/DPCT/APINamesTemplateType.inc

```diff
@@ -38,6 +38,13 @@ TYPE_REWRITE_ENTRY(
     TYPE_FACTORY(STR(MapNames::getDpctNamespace() + "constant_iterator"),
                      TEMPLATE_ARG(0))))
 
+
+TYPE_REWRITE_ENTRY(
+    "cudaExtent",
+    HEADER_INSERTION_FACTORY(HeaderType::HT_DPL_Iterator,
```

Comment:
while it is DPL related?  Please double check it.

Reply:
done